### PR TITLE
Added Badges

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Added 4 types of badges -
   a. Status Badges - Online || Offline || Busy
   b. Icon Badges
   c.  Button with badges
   d. Text with Badges
Any color can be added to the badge icons using the color classes.

Badges

### CODE REVIEW

**Review by ScoobyDubeyDo — APPROVED** [review]
looks fine👍

**Review by talrejah02 — APPROVED** [review]
Everything looks good.

File: styles/components/buttons.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.button {
+  border: transparent;
+  padding: 0.2rem var(--spacing-06);
+  border-radius: var(--border-radius-sm);
+  transition: all 0.4s ease;
+  font-size: 0.9rem;
+}
+
+.button:hover {
+  background-color: rgb(0, 0, 0, 0.2);
```
Comment (talrejah02) [inline styles/components/buttons.css:10]: Use var here also for hover color.

Comment (SukanyaSen530) [reply styles/components/buttons.css:10]: Will update it in next PR :)

**Pull request merged**